include/cpuinfo.h

```diff
@@ -812,6 +812,10 @@ struct cpuinfo_x86_isa {
 	bool avx512vp2intersect;
 	bool avx512_4vnniw;
 	bool avx512_4fmaps;
+	bool amx_bf16;
+	bool amx_tile;
```

Comment:
remove amx_tile?
is tile useful?  all cpus that support amx_bf16 or amx_int8 will support amx_tile, and amd_tile by itself is not useful?

Reply:
I would suggest we keep these flags at the low level that are mapped exactly to underlying CPU ISA feature bits. We can probably have some helper functions like `has_amx_support` at the higher level for ease of use purposes.

Reply:
It is true that the existing platforms that supports amx_bf16 or amx_int8 will support amx_tile. However I prefer to leave a low level flag here, just in case of possible future changes.